src/js/views/forms/form/form_views.js

```diff
@@ -259,21 +260,94 @@ const ReadOnlyView = View.extend({
   template: hbs`{{ @intl.forms.form.formViews.readOnlyView.buttonText }}`,
 });
 
+const SaveButtonTypeDroplist = Droplist.extend({
+  popWidth: 192,
+  align: 'right',
```

Comment:
`align: 'right'` not working was because I had `popWidth` set to `popWidth: '192px'`. Instead of `popWidth: 192`.

For some reason that made the droplist miscalculate the correct position to display itself 🤷 

_This is something Paul and I discussed off of GitHub_.

Reply:
I don't think popWidth is necessary here